gso/.htaccess

```diff
@@ -1,70 +1,109 @@
- 
+Options +FollowSymLinks
+RewriteEngine on 
 
 # Rewrite rule to serve turtle content from the vocabulary URI if requested
- RewriteCond %{HTTP_ACCEPT} text/turtle
-RewriteRule       ^.*\/gso\/ex-complexcontact\/         https://raw.githack.com/Loop3D/GKM/master/Loop3D-GSO/Examples/GSO-ExampleComplexContacts.ttl  [R=303,L]
-RewriteRule       ^.*\/gso\/ex-event1\/         https://raw.githack.com/Loop3D/GKM/master/Loop3D-GSO/Examples/GSO-ExampleEvents1.ttl  [R=303,L]
-RewriteRule       ^.*\/gso\/ex-stratlexicon2-bc\/         https://raw.githack.com/Loop3D/GKM/master/Loop3D-GSO/Examples/GSO-ExampleBritishColumbiaStrat-v2.ttl  [R=303,L]
-RewriteRule       ^.*\/gso\/ex-timelowerjurassic\/         https://raw.githack.com/Loop3D/GKM/master/Loop3D-GSO/Examples/GSO-ExampleEpochLowerJurassic.ttl  [R=303,L]
-RewriteRule       ^.*\/gso\/ex-faults\/         https://raw.githack.com/Loop3D/GKM/master/Loop3D-GSO/Examples/GSO-ExampleFault2.ttl  [R=303,L]
-RewriteRule       ^.*\/gso\/faultKannaV4Model\/         https://raw.githack.com/Loop3D/GKM/master/Loop3D-GSO/Examples/GSO-ExampleFaultKannaV4Model.ttl  [R=303,L]
-RewriteRule       ^.*\/gso\/foldexample\/         https://raw.githack.com/Loop3D/GKM/master/Loop3D-GSO/Examples/GSO-ExampleFold.ttl  [R=303,L]
-RewriteRule       ^.*\/gso\/ex-formationJs\/         https://raw.githack.com/Loop3D/GKM/master/Loop3D-GSO/Examples/GSO-ExampleFormationJs.ttl  [R=303,L]
-RewriteRule       ^.*\/gso\/ex-ausstratunit\/         https://raw.githack.com/Loop3D/GKM/master/Loop3D-GSO/Examples/GSO-ExampleGeosciAustraliaStratUnit.ttl  [R=303,L]
-RewriteRule       ^.*\/gso\/ex-history\/         https://raw.githack.com/Loop3D/GKM/master/Loop3D-GSO/Examples/GSO-ExampleHistory.ttl  [R=303,L]
-RewriteRule       ^.*\/gso\/ex-islewightstrat\/         https://raw.githack.com/Loop3D/GKM/master/Loop3D-GSO/Examples/GSO-ExampleIsleOfWightStrat-pm1.ttl  [R=303,L]
-RewriteRule       ^.*\/gso\/ex-plutontojiza\/         https://raw.githack.com/Loop3D/GKM/master/Loop3D-GSO/Examples/GSO-ExampleLaTojizaPluton.ttl  [R=303,L]
-RewriteRule       ^.*\/gso\/ex-petrophysics-bc\/         https://raw.githack.com/Loop3D/GKM/master/Loop3D-GSO/Examples/GSO-ExamplePetrophysicalProperties_v2.ttl  [R=303,L]
-RewriteRule       ^.*\/gso\/ex-materialCb\/         https://raw.githack.com/Loop3D/GKM/master/Loop3D-GSO/Examples/GSO-ExampleRockMaterialBolsaQuartzite.ttl  [R=303,L]
-RewriteRule       ^.*\/gso\/ex-roles\/         https://raw.githack.com/Loop3D/GKM/master/Loop3D-GSO/Examples/GSO-ExampleRoles.ttl  [R=303,L]
-RewriteRule       ^.*\/gso\/alterationtype-bc\/         https://raw.githack.com/Loop3D/GKM/master/Loop3D-GSO/Examples/GSO-ExampleVocabularyExtension-Alteration_Type-BC.ttl  [R=303,L]
-RewriteRule       ^.*\/gso\/ec-lardeaustrat\/         https://raw.githack.com/Loop3D/GKM/master/Loop3D-GSO/Examples/GSO-LardeauGroup.ttl  [R=303,L]
+RewriteCond %{HTTP:Accept} text/turtle [NC]
```

Comment:
Out of curiosity, why `HTTP:Accept` vs `HTTP_ACCEPT`?  I didn't know until right now that form existed and works on any header.  The "Server-Variable" `HTTP_ACCEPT` version seems much more common.

Reply:
Um, legacy notation from my textbook _The Definitive Guide to Apache mod_rewrite_ perhaps?